api/src/db/migrations/v67_add_graduation_card.ts

```diff
@@ -0,0 +1,294 @@
+import { v66UserData } from "./v66_add_nexus_to_profile";
+
+export interface v67UserData {
+  user: v67BusinessUser;
+  profileData: v67ProfileData;
+  formProgress: v67FormProgress;
+  taskProgress: Record<string, v67TaskProgress>;
+  taskItemChecklist: Record<string, boolean>;
+  licenseData: v67LicenseData | undefined;
+  preferences: v67Preferences;
+  taxFilingData: v67TaxFilingData;
+  formationData: v67FormationData;
+  version: number;
+}
+
+export const migrate_v66_to_v67 = (v66Data: v66UserData): v67UserData => {
```

Comment:
Since there is logic for this migration, this seems like a good place to have a unit test